packages/e2e-tests/utils/account.js

```diff
@@ -0,0 +1,77 @@
+const {
+    connect,
+    keyStores: { InMemoryKeyStore },
+    utils: {
+        KeyPair,
+        format: { parseNearAmount },
+    },
+} = require("near-api-js");
+const { parseSeedPhrase } = require("near-seed-phrase");
+
+const getDefaultConfig = () => ({
+    networkId: "default",
+    nodeUrl: process.env.NODE_URL || "https://rpc.testnet.near.org",
+    walletUrl: process.env.WALLET_URL || "https://wallet.testnet.near.org",
+    keyStore: new InMemoryKeyStore(),
+});
+
+function getKeyPairFromSeedPhrase(seedPhrase) {
+    return KeyPair.fromString(parseSeedPhrase(seedPhrase).secretKey)
+}
+
+function generateTestAccountId() {
+    return `test-account-${Date.now()}-${
+        Math.floor(Math.random() * 1000) % 1000
+    }`;
+}
+
+async function createRandomBankSubAccount() {
+    return await createBankSubAccount(generateTestAccountId());
+}
+
+async function createBankSubAccount(accountId) {
+    const {
+        BANK_ACCOUNT: parentAccountId,
+        BANK_SEED_PHRASE: parentAccountSeedphrase,
+        TEST_ACCOUNT_SEED_PHRASE,
+    } = process.env;
+
+    const testAccountId = `${accountId}.${parentAccountId}`;
+    const testAccountSeedphrase = `${testAccountId} ${TEST_ACCOUNT_SEED_PHRASE}`;
```

Comment:
I don't understand why this includes the `testAccountId`; shouldn't the seed phrase be literally just TEST_ACCOUNT_SEED_PHRASE which would always be 12 words from the word list?

I didn't think seed phrases should have any strings in them other than specific words in it from the word list we use for seed phrases.

Reply:
This is copied from https://github.com/near/near-wallet/blob/master/packages/e2e-tests/index.js#L80 . I assume its to maintain some traceability to the sub accounts. i.e. subaccounts generated will always have their accountId + the parent's seed phrase as their seed phrase.